README.md

```
@@ -14,6 +14,23 @@
+
+Run benchmarks: (you can change the amount of files that will be generated in the bench/query_benchmark.rs)
+``` bash
+cargo bench
+```
+Run flamegraph: (i'd comment out rayon and change `par_iter` to `iter` to reduce the noise in the flamegraph)
+``` bash
+cargo install flamegraph
+cargo flamegraph --root --bin krafna -- 'select file.name, tags from frontmatter_data("../krafna-bench/bench/") where "exampl" in tags'
+```
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)